var/spack/repos/builtin/packages/parthenon/package.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Parthenon(CMakePackage):
+    """A performance portable block-structured adaptive mesh refinement framework."""
+
+    homepage = "https://github.com/parthenon-hpc-lab/parthenon"
+    git = "https://github.com/parthenon-hpc-lab/parthenon.git"
+    url = "https://github.com/parthenon-hpc-lab/parthenon/archive/v0.8.0.tar.gz"
+    maintainers("pbrady", "pgrete")
+
+    version("develop", branch="develop")
+    version("24.03", sha256="ec9109c6bf442237641e627f301567527eb5e756b6959b6747d35315d041727c")
+    version("23.11", sha256="76f79fb7d6556d94052829a8ac71f53cbda76f37fabd9233c5c0cd47ef561aee")
+    version("0.8.0", sha256="9ed7c9ebdc84927a43b86c1e061f925b57cef9b567c7275f22779ed4d98e858d")
+
+    # ------------------------------------------------------------#
+    # Variants
+    # ------------------------------------------------------------#
+
+    variant("single", default=False, description="Run in single precision")
+    variant("mpi", default=True, description="Enable mpi")
+    variant(
+        "host_comm_buffers", default=False, description="Allocate communication buffers on host"
+    )
+    variant("hdf5", default=True, description="Enable hdf5")
+    variant(
```

Comment:
It looks like this should be inside a `with when("+hdf5")` like:

```
with when("+hdf5"):
    variant(
        "compression", default=True, description="Enable compression in hdf5 output/restart files"
    )
```

since it looks like this variant only applies when you have `+hdf5`

Reply:
Thanks for the explanation.  Package has been updated